publisher-sdk/src/main/java/com/criteo/publisher/InHouse.java

```diff
@@ -61,14 +62,39 @@ public void loadBidResponse(
       return;
     }
 
-    CdbResponseSlot slot = bidManager.getBidForAdUnitAndPrefetch(adUnit);
-    if (slot == null) {
-      bidResponseListener.onResponse(null);
-      return;
+    bidManager.getBidForAdUnit(adUnit, new BidResponseMapper(adUnit, bidResponseListener, clock));
+  }
+
+  private static class BidResponseMapper implements BidListener {
```

Comment:
I suggest renaming to `CdbResponseToBidResponseMapper`. Though not s ure why this can't just be a method instead of a class?

Reply:
`BidResponse` will be renamed to `Bid` and will be the object used by both AppBidding and InHouse. Hence the InHouse class will disappear soon (has it has no more logic).

By method, do you mean anonymous class ? If yes I could do that.

Reply:
Yep seems like an anonymous class can just do the job simply.

Reply:
Done